dataset.go

```diff
@@ -38,12 +38,8 @@ func (m *DataSet) FieldByType(dsType DataSetType) (interface{}, error) {
 		return nil, ErrNilDataSet
 	}
 
-	switch dsType {
-	case DataSetType_KEYWORD, DataSetType_SENTIMENT:
-		if m.Keyword == nil {
-			return nil, nil
-		}
-		return m.Keyword, nil
+	if dsType == DataSetType_SENTIMENT {
+		name = DataSetType_name[int32(DataSetType_KEYWORD)]
 	}
```

Comment:
I'm still not too thrilled about having code that's doing reflection have types hardcoded in, but seeing the proto changes that were made, this was my solution to at least simplify it a little. I'd appreciate other team members' input. @joshuarubin @drekar @michaeltrobinson